extensions/guacamole-auth-radius/src/main/resources/guac-manifest.json

```diff
@@ -11,7 +11,8 @@
 
     "translations" : [
         "translations/en.json",
-        "translations/ja.json"
+        "translations/ja.json",
+        "translations/de.json"
```

Comment:
Please maintain alphabetical order of the filenames listed under `translations` for any particular `guac-manifest.json`.